externs/browser/webgl2_compute.js

```diff
@@ -0,0 +1,1608 @@
+/*
+ * Copyright 2019 The Closure Compiler Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Definitions for WebGL functions as described at
+ * http://www.khronos.org/registry/webgl/specs/latest/
+ *
+ * This file is current up to the WebGL 2.0 Compute spec.
+ *
+ * This relies on webgl2.js and html5.js being included for WebGL2, Canvas and
+ * Typed Array support.
+ *
+ * @externs
+ */
+
+
+/**
+ * @constructor
+ * @extends {WebGL2RenderingContext}
```

Comment:
Could you also link to https://www.khronos.org/registry/webgl/specs/latest/2.0-compute/#webgl2-compute-context here?

Reply:
Done. But isn't it duplicated with the `@fileoverview` part? I didn't see a similar line in `webgl2.js`.

Reply:
For me, it's helpful to have a link pointing directly to the `WebGL2ComputeRenderingContext` section, as well as to the spec as a whole. I acknowledge that's a matter of preference, though.